alvr/dashboard/src/data_sources.rs

```diff
@@ -103,7 +103,10 @@ impl DataSources {
 
                 while running.value() {
                     while let Ok(request) = requests_receiver.try_recv() {
-                        debug!("Dashboard request: {request:?}");
+                        debug!(
```

Comment:
This should not be under some debug group, so it can be disabled because this seems really annoying. Or rather under the raw events or spammy events stuff.

Reply:
This is the dashboard, it doesn't have the debug groups. In release builds, debug log is not shown

Reply:
I think we should come back to this after the patch release.

Reply:
I think dashboard requests might be fine to log, all things considered, but I'm really not sure about server events, those seem insanely spammy and I really want to keep logs a bit clean so you can actually find relevant messages

Reply:
This is just for debug builds. There is a need for this from developers using ALVR to discover the dashboard API, because we do automatic serde serialization so the api can't be found anywhere in the code

Reply:
I'll try out a build with this and see how bad it actually is and judge whether there I'll request a dedicated toggle for this from you

Reply:
I've yet to try out a build, let's just merge it and we can still revert it, sorry for this taking so long